File: C#/백준/Bronze/6162. Superlatives/Superlatives.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int t = int.Parse(Console.ReadLine());
            for (int i=0;i<t;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int a = aa[0]; int b = aa[1];
                Console.WriteLine($"Data Set {i+1}:");
                if (a<=b){
                    Console.WriteLine("no drought\n");
                }
                else{
                    String s = "";
                    while(true){
                        if (a>b && a<=b*5){
                            s+="drought\n";
                            break;
                        }
                        else{
                            s+="mega ";
                            b*=5;
                        }
                    }
                    Console.WriteLine(s);
                }
            }
        }
    }
}

File: Python/백준/Bronze/6162. Superlatives/Superlatives.py
t = int(input())
for i in range(t):
    a,b = map(int,input().split())
    print(f'Data Set {i+1}:')
    if (a<=b):
        print('no drought\n')
    else:
        n=1
        s = ''
        while(True):
            if (b*n<a and a<=5*b*n):
                s+='drought'
                break
            else:
                s+='mega '
                n*=5
        print(s)
        print()